packages/editor-ui/src/components/ParameterInput.vue

```diff
@@ -585,6 +585,21 @@ export default mixins(
 			closeExpressionEditDialog () {
 				this.expressionEditDialogVisible = false;
 			},
+			trackExpressionEditOpen () {
+				if(!this.node) {
+					return;
+				}
+
+				const nodeTypeSplit = this.node.type.split('.');
+				if(nodeTypeSplit.length === 2 && nodeTypeSplit[0] === 'n8n-nodes-base') {
```

Comment:
Maybe we could also track as `Custom node`, WDYT? This is an interesting number to have.